pages/[[...slug]]/index.tsx

```diff
@@ -29,10 +28,12 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 }
 
 export const getStaticPaths: GetStaticPaths = async () => {
-  // allPageContentsWithSlugString = [ { slugString: 'demo/js' }, { slugString: 'demo/python' } ]
+  type Path = { params: { slug: string[] } }
+
   const allPageContentsWithSlugString: PageContentsWithSlugString[] = await getAllPageContentsWithSlugString()
-  // paths: [{ params: { slug: ['demo', 'js'] } }, { params: { slug: ['demo', 'python'] } }],
-  let paths = allPageContentsWithSlugString.map(({ slugString }) => ({ params: { slug: slugString.split('/') } }))
```

Comment:
I deleted the comment in the code. 
I see what's the problem. Probably because there are too many commits in this PR which makes it very hard to track the changes in the 'File changed' tab. Sorry for making this inconvenience. 
Anyway, I will push further commits to deal with the unsolved problems.